Privacy.md

```diff
@@ -0,0 +1,88 @@
+# Forecastie
+
+# Privacy Policy
+
+This privacy policy statement is created to clearly state the developer's policies with respect to data collection, usage, and security. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized, and the choices you have associated with that data.
+If you choose to use this Service, then you agree to the collection and use of information in relation to this policy. The Personal Information that may be collected is used for providing and improving the Service. This information will not be used or shared with anyone as described in this Privacy Policy.
+Kindly note that this Privacy Policy applies only to this software. The developer would not be held responsible for the content or privacy practices on any third party app not operated by the developer to which this software links, or that links to this software. Therefore, it is advisable to read up the privacy policies of third party services that may be linked to this software.
```

Comment:
GPLv3 expressly states there isn't any. See 15. 16. and 17. here https://www.gnu.org/licenses/gpl-3.0.html
You also can't put up alternate claims/licenses, as this is not permitted. So why? Keep it short and sweet.